package_trinity_2_1_1 fails with /usr/bin/ld: cannot find -lblas

We tried adding LD_LIBRARY_PATH variable to lapack dependency but the issue still persists, any ideas.

Full error message here:

    * installing *source* package ‘RcppArmadillo’ ...
    ** package ‘RcppArmadillo’ successfully unpacked and MD5 sums checked
    ** libs
    /usr/bin/ld: cannot find -lblas
    collect2: ld returned 1 exit status
    make: *** [RcppArmadillo.so] Error 1
    ERROR: compilation failed for package ‘RcppArmadillo’
    * removing ‘/galaxy/production/tool_dependencies/trinity/2.1.1/iuc/package_trinity_2_1_1/f75117647d9c/RcppArmadillo’

--- a/packages/package_lapack_3_4/tool_dependencies.xml
+++ b/packages/package_lapack_3_4/tool_dependencies.xml
@@ -14,7 +14,7 @@
                     cd build &&
                     cmake .. -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR/lapack -DCMAKE_Fortran_FLAGS='-O2 -fPIC' &&
                     make &&
-                    make install
+                    make install &&
                     cmake .. -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR/lapack -DBUILD_SHARED_LIBS=ON -DCMAKE_Fortran_FLAGS='-O2 -fPIC' &&
                     make &&
                     make install

--- a/packages/package_lapack_3_4/tool_dependencies.xml
+++ b/packages/package_lapack_3_4/tool_dependencies.xml
@@ -8,21 +8,20 @@
             <actions>
                 <action type="download_by_url" sha256sum="60a65daaf16ec315034675942618a2230521ea7adf85eea788ee54841072faf0">https://depot.galaxyproject.org/software/lapack/lapack_3.4.2_src_all.tar.gz</action>
 
-                <action type="shell_command">mkdir build</action>
-                <action type="shell_command">
-                    cd build &amp;&amp;
-                    cmake .. -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR/lapack -DCMAKE_Fortran_FLAGS='-O2 -fPIC'
-                </action>
-
+                <!-- compile both static and shared library -->
                 <action type="shell_command"><![CDATA[
                     mkdir build &&
                     cd build &&
                     cmake .. -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR/lapack -DCMAKE_Fortran_FLAGS='-O2 -fPIC' &&
                     make &&
                     make install
+                    cmake .. -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR/lapack -DBUILD_SHARED_LIBS=ON -DCMAKE_Fortran_FLAGS='-O2 -fPIC' &&
+                    make &&
+                    make install
                     ]]></action>
                 <action type="set_environment">
                     <environment_variable name="LAPACK_LIB_DIR" action="set_to">$INSTALL_DIR/lapack/lib</environment_variable>
+                    <environment_variable name="LD_LIBRARY_PATH" action="prepend_to">$INSTALL_DIR/lapack/lib</environment_variable>
                 </action>
             </actions>
         </install>

